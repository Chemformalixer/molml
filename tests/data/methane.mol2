@<TRIPOS>MOLECULE
qmxyz/qm-0000.out
 5 4 0 0 0
SMALL
GASTEIGER

@<TRIPOS>ATOM
      1 C           0.9983   -0.0025   -0.0044 C.3     1  LIG1       -0.0776
      2 H           2.0902   -0.0024    0.0041 H       1  LIG1        0.0194
      3 H           0.6338    1.0269    0.0041 H       1  LIG1        0.0194
      4 H           0.6270   -0.5277    0.8781 H       1  LIG1        0.0194
      5 H           0.6414   -0.5075   -0.9054 H       1  LIG1        0.0194
@<TRIPOS>BOND
     1     5     1    1
     2     1     2    1
     3     1     3    1
     4     1     4    1
