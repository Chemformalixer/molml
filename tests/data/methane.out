C 0.99826008 -0.00246000 -0.00436000
H 2.09021016 -0.00243000 0.00414000
H 0.63379005 1.02686007 0.00414000
H 0.62704006 -0.52773003 0.87811010
H 0.64136006 -0.50747003 -0.90540005
